app/src/main/java/com/alphawallet/app/entity/TokensMapping.java

```diff
@@ -0,0 +1,59 @@
+package com.alphawallet.app.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class TokensMapping {
+
+    public class Contract {
+
+        @SerializedName("address")
+        @Expose
```

Comment:
@Expose is also not required.
![image](https://user-images.githubusercontent.com/735708/147894444-c6cc770e-1601-40a3-91b6-65876dfd49ba.png)